src/libs/antares/study/include/antares/study/parts/hydro/container.h

```diff
@@ -28,6 +28,63 @@
 
 namespace Antares::Data
 {
+
+enum dayYearCount
+{
+    //! The maximum number of days in a year
+    dayYearCount = 366
+};
+
+struct HydroDaily
+{
+    //! Net demand, for each day of the year, for each area
+    std::array<double, dayYearCount> DLN;
+    //! Daily local effective load
+    std::array<double, dayYearCount> DLE;
+};
+
+struct HydroMonthly
+{
+    //! Monthly local effective demand
+    std::array<double, 12> MLE;
```

Comment:
You may use the comment above to rename these variables. Quite unexpressive right now